solidity/contracts/bancorx/BancorX.sol

```diff
@@ -431,7 +434,10 @@ contract BancorX is IBancorX, TokenHolder, ContractRegistryClient {
     function getCurrentLockLimit() public view returns (uint256) {
         // prevLockLimit + ((currBlockNumber - prevLockBlockNumber) * limitIncPerBlock)
         uint256 currentLockLimit = prevLockLimit.add(((block.number).sub(prevLockBlockNumber)).mul(limitIncPerBlock));
-        if (currentLockLimit > maxLockLimit) return maxLockLimit;
+        if (currentLockLimit > maxLockLimit) {
```

Comment:
Can simplify this to:
```
return Math.min(currentLockLimit, maxLockLimit);
```

Reply:
This PR is already too big, so let's postpone unrelated old code refactoring (unlike style or formatting)